src/WrappedVault.sol

```diff
@@ -244,14 +244,15 @@ contract WrappedVault is Owned, ERC20, IWrappedVault {
 
         uint256 remainingRewards = rewardsInterval.rate * (rewardsInterval.end - newStart);
         uint256 rate = (rewardsAfterFee + remainingRewards) / (newEnd - newStart);
```

Comment:
Since you no longer re-use `rewardsAfterFee` below, you could eliminate its declaration above and inline its computation here:

```solidity
        uint256 rate = (rewardsAdded - frontendFeeTaken - protocolFeeTaken + remainingRewards) / (newEnd - newStart);
```